### ISSUE TITLE
What happened to the repository?

### ISSUE THREAD

**Issue by quazar-omega** [unicode-table/unicode-table-data#214]
What's the reasoning behind cf2a9ce8e0ccfdb5ccbfd00f4aa2b85195a3ca70?  
This is a good source of data on characters IMO, why delete everything all of a sudden? Ironically, as a user, I feel mislead *now* rather than before 🤔

**Comment by asanov** [comment]
Irrelevant and incorrect data and translations has been removed so as not to mislead users.

**Comment by quazar-omega** [comment]
Yes I read the commit message, but I don't get it, are you going to put new data in another format in the future?

**Comment by asanov** [comment]
We will only do it when we are completely confident in the quality of this data and the accuracy of the translations. At this time, to avoid errors, we recommend using the official information provided by the Unicode Consortium.

**Comment by quazar-omega** [comment]
I see, though, rather than deleting all files, it might be better to leave them up as is and instead archive the repository with a notice in the README? Your response sounds like a good fit for the text of the notice
